managed/models/artifact_helpers.go

```diff
@@ -67,6 +69,13 @@ func FindArtifacts(q *reform.Querier, filters ArtifactFilters) ([]*Artifact, err
 	if filters.Status != "" {
 		conditions = append(conditions, fmt.Sprintf("status = %s", q.Placeholder(idx)))
 		args = append(args, filters.Status)
+		idx++
+	}
+
+	if filters.Folder != nil {
+		conditions = append(conditions, fmt.Sprintf("folder = %s", q.Placeholder(idx)))
+		args = append(args, *filters.Folder)
+		// idx++
```

Comment:
Leftover?

Reply:
Nope, I leave it to not forget uncomment it if adding more fields to filter.